docs/releases/2.0.txt

```diff
@@ -76,6 +76,11 @@ Minor features
   :class:`~django.contrib.postgres.aggregates.ArrayAgg` determines if
   concatenated values will be distinct.
 
+* Added the :class:`~django.contrib.postgres.functions.RandomUUID` database
```

Comment:
I think this deserves 2 bullet points. One for `RandomUUID` and one for `pgcrypto`.